src/config/Config.zig

```diff
@@ -4256,6 +4256,33 @@ pub const RepeatablePath = struct {
             var buf: [std.fs.max_path_bytes]u8 = undefined;
             const abs = dir.realpath(path, &buf) catch |err| abs: {
                 if (err == error.FileNotFound) {
+                    // Check if the path starts with a tilde and expand it to the home directory on linux/mac
+                    if (path[0] == '~') {
+                        const home_env_var = switch (builtin.os.tag) {
+                            .linux, .macos => std.posix.getenv("HOME"),
+                            .windows => null,
+                            else => null,
+                        };
+                        if (home_env_var) |home_dir| {
```

Comment:
Instead of directly reading the `HOME` env var, there's a function called `home` in `src/os/homedir.zig` that should be re-used. It works on more platforms and also handles cases where `HOME` is not set by checking in other platform-specific places for a user's home directory.